acceptance/Rakefile

```diff
@@ -78,6 +78,11 @@ def beaker_test(mode = :packages, options = {})
     final_options[:install] << "#{build_giturl('hiera')}##{sha}"
   end
 
+  if mode == :peaio
+    final_options[:puppet_agent_sha] = sha
+    final_options[:puppet_agent_version] = sha
+  end
+
```

Comment:
It maps to agent-download versions (agent-downloads.delivery...), which seem to have their own convention - puppet-agent releases with an additional tag, like `1.2.1.41`.